guides/source/action_mailbox_basics.md

```diff
@@ -324,9 +324,10 @@ end
 
 ## Incineration of InboundEmails
 
-By default, an InboundEmail that has been successfully processed will be
-incinerated after 30 days. This ensures you're not holding on to people's data
-willy-nilly after they may have canceled their accounts or deleted their
+By default, an InboundEmail that has been processed will be
+incinerated after 30 days, regardless of whether the procession succeeds or fails.
```

Comment:
I found that incineration happens regardless of whether the process is successful or not. So, I tried to describe it more clearly.

https://github.com/rails/rails/blob/6f6c211f4781e7bda63e5f47089f9d2612dc0a52/actionmailbox/app/models/action_mailbox/inbound_email/incineratable.rb#L10C5-L10C24